addOns/oast/src/main/java/org/zaproxy/addon/oast/services/interactsh/InteractshService.java

```diff
@@ -147,7 +147,7 @@ public void optionsChanged(OptionsParam optionsParam) {
             try {
                 deregister();
                 register();
-                LOGGER.debug(
+                LOGGER.info(
                         "Updated Interactsh params - server URL: {} and polling frequency: {} seconds.",
```

Comment:
I think it's worth logging that it was changed (and letting the user know, assuming they look at logs/console).

Reply:
Why so if the user is changing it itself?